src/data/support.lean

```diff
@@ -146,6 +146,10 @@ lemma support_comp_eq [has_zero A] [has_zero B] (g : A → B) (hg : ∀ {x}, g x
   support (g ∘ f) = support f :=
 set.ext $ λ x, not_congr hg
 
+lemma support_comp_eq_preimage [has_zero B] (g : A → B) (f : α → A) :
+  support (g ∘ f) = f ⁻¹' support g :=
+rfl
```

Comment:
This one is already in master from another PR of yours, if I am not mistaken. Anyway, this shouldn't cause a conflict since the lines are exactly the same.